apps/desktop/.eslintignore

```diff
@@ -0,0 +1,2 @@
+.eslintrc.*
```

Comment:
Hmm, why?

Reply:
It's a strait-forward solution to prevent ESlint from such kind of issues. Could be solved in a more elegant way in the future
<img width="740" alt="image" src="https://github.com/trilitech/umami-v2/assets/169451151/e793b21f-91d7-4b10-8010-feb7d868b0f8">